app/actions/websocket/channels.test.js

```diff
@@ -4,24 +4,23 @@
 /* eslint-disable no-import-assign */
 
 import assert from 'assert';
-import nock from 'nock';
+
 import {Server, WebSocket as MockWebSocket} from 'mock-socket';
-import thunk from 'redux-thunk';
+import nock from 'nock';
 import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import TestHelper from 'test/test_helper';
```

Comment:
I think the test helpers should also be under a directory `@test`

Reply:
As far as I know, there is no `@test` alias so far in the project.